code/modules/research/designs/AI_module_designs.dm

```diff
@@ -8,7 +8,7 @@
 	id = "freeform_module"
 	req_tech = list("programming" = 5, "materials" = 4)
 	build_type = IMPRINTER
-	materials = list(MAT_GLASS = 1000, "sacid" = 20, MAT_GOLD = 100)
+	materials = list(MAT_GLASS = 1000, MAT_GOLD = 100)
```

Comment:
You removed the acid requirement entirely?

Reply:
Yes, as I documented in the post, I removed acid requirements from everything; for the most part, it's pointless busywork, and not an actual gate like, say, gold, diamond, or bluespace mesh is.

Reply:
Addressed the inconsistency of these two parts; their base versions required acid, but their advanced versions did not; now, none of them do.